### PR TITLE
Added a features section the readme

### PR DESCRIPTION
**Pull request by LadyNaggaga**
We were missing some of our key features on our readme page. I also updated our intro image to show VS Code instead of insiders. 
Please changes below

newnotebook

### Multi-language notebooks 
.NET Interactive enables users to mix languages in a single notebook or cell without a wrapper. The multi-language experience opens up doors for users to use the best language for the task at hand.

**Languages supported**
- C# and F# 
- PowerShell built in collaboration with the PowerShell team 💙
- JavaScript
- HTML 
- SQL built in collaboration with the  Azure Data / SQL team 💙

**Coming soon**
- Kusto 

**What languages are we exploring** 
- Python and R 

### Variable Sharing 

.NET Interactive enables you to write code in multiple languages within a single notebook and in order to take advantage of those languages' different strengths, you might find it useful to share data between them. Read more [here](https://github.com/dotnet/interactive/blob/main/docs/variable-sharing.md).

Notebooks-variable-sharing

The gif above showcases the following:
- Variable sharing across C#, HTML and JavaScript across cells.
- Multi-language cells.

For more examples on multi-language notebooks and variable sharing checkout our [polyglot samples](https://github.com/dotnet/interactive/tree/main/samples/notebooks/polyglot).

### Visualization 

**Low code visualization**

In just a single line of code easily visualize data with Microsoft SandDance and [nteract DataExplorer](https://data-explorer.nteract.io/). For  example the code snippet below will render an interactive [Microsoft SandDance](https://www.microsoft.com/en-us/research/project/sanddance/).
```csharp
housingData.ExploreWithSandDance().Display();
```

low-code-visualization

For more [low code visualization](https://github.com/dotnet/interactive/tree/main/samples/ExtensionLab) example check out our samples.

**Works with your favorite visualization libraries**

d3js

The image below showcases the following: 

- `C#` cell: Define a variable in C#
- `JavaScript` cell: Use RequireJS to import d3.js
- `HTML cell`:  Visualize the data

Full example [here](https://github.com/dotnet/interactive/blob/main/samples/notebooks/polyglot/d3js.ipynb).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +9,7 @@
 * [Code bots](https://github.com/CodeConversations/CodeConversations)
 * Devices like [Raspberry Pi](https://www.raspberrypi.org/)
 * Embeddable script engines
-* REPLs
+* [REPLs](https://github.com/jonsequitur/dotnet-repl)
```
Comment (jonsequitur) [inline README.md:12]: 😊

File: README.md [hunk 2]
```diff
@@ -18,7 +19,65 @@
...
+- C# and F# 
+- PowerShell built in collaboration with the PowerShell team 💙
+- JavaScript
+- HTML 
+- SQL built in collaboration with the  Azure Data / SQL team 💙
+
+**Coming soon**
+- Kusto 
+
+**What languages are we exploring**
```
Comment (jonsequitur) [inline README.md:40]:
```suggestion
**What languages are we exploring?** 
```

File: README.md [hunk 3]
```diff
@@ -18,7 +19,65 @@
...
+- Python and R 
+
+### Variable Sharing 
+
+.NET Interactive enables you to write code in multiple languages within a single notebook and in order to take advantage of those languages' different strengths, you might find it useful to share data between them. Read more [here](https://github.com/dotnet/interactive/blob/main/docs/variable-sharing.md).
+
+![Notebooks-variable-sharing](https://user-images.githubusercontent.com/2546640/130664292-1cdfb806-a6f6-4874-bcad-a5eb4517a925.gif)
+
+The gif above showcases the following:
+- Variable sharing across C#, HTML and JavaScript across cells.
```
Comment (jonsequitur) [inline README.md:50]:
```suggestion
- Variable sharing across C#, HTML and JavaScript cells.
```

File: README.md [hunk 4]
```diff
@@ -18,7 +19,65 @@
...
+- Multi-language cells.
+
+For more examples on multi-language notebooks and variable sharing checkout our [polyglot samples](https://github.com/dotnet/interactive/tree/main/samples/notebooks/polyglot).
```
Comment (jonsequitur) [inline README.md:53]:
```suggestion
For more examples on multi-language notebooks and variable sharing check out our [polyglot samples](https://github.com/dotnet/interactive/tree/main/samples/notebooks/polyglot).
```

File: README.md [hunk 5]
```diff
@@ -18,7 +19,65 @@
...
+**Low code visualization**
+
+In just a single line of code easily visualize data with Microsoft SandDance and [nteract DataExplorer](https://data-explorer.nteract.io/). For  example the code snippet below will render an interactive [Microsoft SandDance](https://www.microsoft.com/en-us/research/project/sanddance/).
+```csharp
+housingData.ExploreWithSandDance().Display();
+```
+
+![low-code-visualization](https://user-images.githubusercontent.com/2546640/130510820-6a5b5f9d-a0cc-4fef-8a3d-ea741a30d7f8.gif)
+
+For more [low code visualization](https://github.com/dotnet/interactive/tree/main/samples/ExtensionLab) example check out our samples.
```
Comment (jonsequitur) [inline README.md:66]:
```suggestion
For more [low code visualization](https://github.com/dotnet/interactive/tree/main/samples/ExtensionLab) examples, check out our samples.
```

File: README.md (hunk 1 above)
Comment (LadyNaggaga) [reply README.md:12]: Should we add a section on it?

Comment (jonsequitur) [reply README.md:12]: No strong opinion.

**Review by jonsequitur — APPROVED** [review]

**Pull request merged**